src/main/java/io/stargate/sgv2/jsonapi/service/bridge/executor/NamespaceCache.java

```diff
@@ -99,10 +105,26 @@ private Uni<CollectionProperty> getVectorProperties(String collectionName) {
                                   .get(DocumentConstants.Fields.VECTOR_INDEX_FUNCTION_NAME));
                     }
                   }
-                  return new CollectionProperty(vectorEnabled, function);
+                  final String comment = table.get().getOptionsOrDefault("comment", null);
+                  CreateCollectionCommand.Options.VectorizeConfig vectorizeConfig = null;
+                  if (comment != null && !comment.isBlank()) {
+                    try {
+                      vectorizeConfig =
+                          objectMapper.readValue(
+                              comment, CreateCollectionCommand.Options.VectorizeConfig.class);
```

Comment:
As per my note next to class, I think we should use a general CollectionConfiguration object and have `VectorizeConfig` be main-level property. This allows easier extension of configuration that we will probably need -- f.ex for Text tokenization, likely.

Reply:
Made the code to resolve the values using JsonNode, so the class need not be moved.